app/views/shared/_header.html.erb

```diff
@@ -1,22 +1,22 @@
 <nav>
   <div id ='nav' class="navbar">
-<!-- Menu hamburguesa-->
+    <!-- Menu hamburguesa-->
     <div class="navbar-hamburguer navbar-mr">
       <button id='menu'>
         <i class="fas fa-bars"></i>
       </button>
     </div>
-<!-- Menu hamburguesa-->
-
-<!-- Barra de busqueda-->
-    <input class="input-field flex-grow navbar-mr-input" type="sarch" name="search" placeholder="Escribe tu dirección">
-<!-- Barra de busqueda-->
-
- <!-- Logo -->
+    <!-- Menu hamburguesa-->
+    <!-- Barra de busqueda-->
+    <div class="pt-1 pl-6 relative mx-0 text-gray-600 grid grid-cols-3 w-10/12">
+      <input class="input-field flex-grow navbar-mr-input type="search" name="search" placeholder="Escribe tu dirección" data-maps-target="field" data-action="keydown->maps#preventSubmit" >
```

Comment:
If you could use the classes that were previously added in the input field it would be awesome.
The purpose of those classes is to standardize the look of the app and reuse as much as possible styles.
Also, if there is needed a button for getting the current location, send me a ss of it to take it into consideration on the design 😃